data-access/src/functions/http-graphql/schema/types/vendor-user.resolvers.ts

```diff
@@ -0,0 +1,29 @@
+// vendor-user.resolvers.js
+import { isValidObjectId } from 'mongoose';
+import { Resolvers } from '../builder/generated';
+
+
+const vendorUserResolvers: Resolvers = {
```

Comment:
Refer to user.resolvers.ts for a reference on the vendor user graphql resolvers. But also keep vendorUserUpdate and have it call context.applicationServices.users.vendorUser.domainApi.updateVendorUser(args.input);